httputil/httputil.go

```diff
@@ -153,8 +183,22 @@ func DownloadBinary(originURL, destDir, destFile string) (string, error) {
 			}
 		}()
 
+		u, err := url.Parse(originURL)
+		if err != nil {
+			// originURL supposed to be valid
+			panic(err)
+		}
+
 		log.Printf("Downloading %s...", originURL)
-		resp, err := get(originURL, "")
+
+		var auth string = ""
+		t, err := tryFindNetrcFileCreds(u.Host)
+		if err == nil {
+			// successfully parsed netrc for given host
```

Comment:
I'd fail here if the .netrc file exists, but could not be read (for whatever reasons).

Reply:
It would also be good to add a log statement regarding whether a .netrc file was read or not

Reply:
>I'd fail here if the .netrc file exists, but could not be read (for whatever reasons).

.netrc may exist, but may not contain given "Host". In this case we should proceed with normal auth, without a failure, because remote may not have auth at all.